addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -335,7 +335,7 @@ def _compute_from_holiday_status_id(self):
 
     @api.depends("allocation_type", "holiday_status_id", "accrual_plan_id")
     def _compute_type_request_unit(self):
-        for allocation in self:
+        for allocation in self.sudo():
```

Comment:
why won't we do` allocation.sudo().accrual_plan_id.added_value_type` instead? This way it's also apparent what is sudo needed for.

Also while you are at it, you can also check/fix commit message, there are some typos

Reply:
@sofiagvaladze thank you for the feedback, I updated the PR :ok_hand: